src/components/data-management/ProjectDetails.jsx

```diff
@@ -36,28 +73,40 @@ const ProjectDetails = ({ width, height }) => {
             </Button>
             <ProjectMenu />
           </Space>
-        </Row>
-        <Row>
-          <Col>
-            <Space direction='vertical' size='small'>
-              <Text type='secondary'>{`ID : ${activeProjectUuid}`}</Text>
-              <Text strong>Description:</Text>
-              <Paragraph
-                editable={{
-                  onChange: (description) => dispatch(
-                    updateProject(activeProjectUuid, { description }),
-                  ),
-                }}
-              >
-                {activeProject.description}
-              </Paragraph>
-            </Space>
-          </Col>
-        </Row>
-        <SamplesTable
-          height={height}
-          ref={samplesTableRef}
-        />
+        </div>
+        <div>
+          <Text type='secondary'>
+            {`Project ID: ${activeProjectUuid}`}
+            {' - '}
+            {`Experiment ID: ${activeProject.experiments[0]}`}
```

Comment:
Added `experimentId` so it's easier for users to see what experiment is associated with the project.